posthog/models/organization.py

```diff
@@ -129,13 +129,16 @@ class OrganizationInvite(UUIDModel):
         related_query_name="organization_invite",
         null=True,
     )
+    emailing_attempt_made: models.BooleanField = models.BooleanField(default=False)
```

Comment:
GQ. Maybe it's better to use the already existing `MessagingRecord`? I agree this is pretty straightforward, but it might be better to use the same module for the same logic?

Reply:
This field is just so that the frontend can determine if it should say that the invite was emailed, but if we're worried about duplicate emails than definitely need to use `MessagingRecord` (or figure out a different fix 🤔).